common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -464,6 +524,21 @@ innovation_troll_tactics = {
 	flag = global_maa
 }
 
+innovation_tuskarr_tactics = {
```

Comment:
What about drylands?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155878829-e147c085-b76a-4f52-8b1a-2c02e3de9efa.png)

</details>